content/updates/03-2020/1-rules-engine-extended-capabilities.md

```diff
@@ -0,0 +1,15 @@
+# Rules engine extended capabilities
+::: meta-data type="Improvement" features="Rules engine, Productivity, Automation" available="19th of March"
+:::
+
+## A new rule action to concatenate attributes
+A new `concatenate` rule action is now available and will certainly boost your rules engine usage! This new action will enable you to **concatenate two attribute values together or more**, and set this new value in another attribute.   
+For instance, you want to make sure your product titles are always formatted the same way. Create a rule to concatenate your product attributes `name` + `color` + `size` and set this new value chain into your `product title` attribute. Use this rule to enhance your product content!
+
+## More automation with the `copy` action
+Ease and faster your product enrichment thanks to the extended capabilities of the `copy` action from our rules engine! You can now **copy the value from attribute type to another attribute type**. For instance, create a rule to copy an option from a simple select attribute to a reference entity single link, instead of manually copying/pasting it. By automating the copy of values, **you make sure you have the right data at the right place**, this will take out some manual and error-prone tasks.
```

Comment:
```suggestion
Ease and speed up your product enrichment thanks to the extended capabilities of the `copy` action from our rules engine! You can now **copy the value from an attribute type to another attribute type**. For instance, you can create a rule to copy an option from a simple select attribute to a reference entity single link, instead of manually copy/pasting it. By automating the copy of values, **you make sure you have the right data in the right place**, this will get rid of some manual and error-prone tasks.
```